src/index.ts

```diff
@@ -227,16 +230,18 @@ function getHandlers(
 
   const onEnd = (event: HandledEvents) => {
     set((state, props) => {
-      let eventData: EventData | undefined;
-      if (state.swiping) {
-        eventData = { ...state.eventData, event } as EventData;
+      //let eventData: EventData | undefined;
+      const eventData = { ...state.eventData, event } as EventData;
 
+      if (state.swiping) {
```

Comment:
❌ put back.
- we do not want to save the `eventData` if it was not a swipe

Reply:
put things back to how they were and left the else statement